### PR TITLE
Should have worked?

### PR DESCRIPTION
**Pull request by mmarchetti2**
I'm basically uploading this because I'm not quite sure what to do next.
I thought it would have worked, but I guess not.

### CODE REVIEW

**Comment by mmarchetti2** [comment]
Ooooh, wait, no, I get it. Here we go.

**Comment by mmarchetti2** [comment]
Oh, okay, I'm guessing it has the same problem as last time. I'll upload it with the folder and edited picture files.

**Comment by mmarchetti2** [comment]
Huh, well I just checked the build, and it says that the 'testforOctocat' failed. Is that to check if the octocat actually was on the pictures? Because it worked fine on my computer. Also, it seems like a lot of other people are having the same problem.

File: resizeAndAddLogo.py [hunk 1]
```diff
@@ -8,7 +8,27 @@
...
+    im = Image.open(os.path.join('originals', filename))
+    width, height = im.size
+    if width > SQUARE_FIT_SIZE and height > SQUARE_FIT_SIZE:
+        if width > height:
+            height = int((SQUARE_FIT_SIZE / width) * height)
+            width = SQUARE_FIT_SIZE
+        else:
+            width = int((SQUARE_FIT_SIZE / height) * width)
+            height = SQUARE_FIT_SIZE
+    print('Resizing %s...' % (filename))
```
Comment (halfaro1) [inline resizeAndAddLogo.py]: @mmarchetti2 shouldn't this resize happen as part of the previous `if` block? The one in line 23.

**Comment by halfaro1** [comment]
@mmarchetti2 there are a series of tests running, one of those tests is whether the Octocat is in the correct place, on a correctly sized image. Thanks for the heads up on lots of folks having issues. I see that and I'm working through giving folks some feedback. In your case, try what I suggested above and see if that can get it to work. It should also explain why the tests aren't passing.

**Comment by mmarchetti2** [comment]
Huh, well I changed it, and it seems it failed again.

**Comment by halfaro1** [comment]
@mmarchetti2 
The automated testing for this particular assignment isn't working as expected, so I've graded your project manually. Everything is 👍 on your project, so I am closing this and changing the tag to `graded`. When I transfer these grades to Blackboard, you'll see the tag switch to `grade updated on blackboard`. Great job!

**Pull request closed without merging**